source/index.html.md

```diff
@@ -560,8 +560,8 @@ payment method must be omitted.
 
 <p class="alert alert-info">
   <b>Notice:</b> An approved authorization that includes
-  <code>card[pares]</code> is 3-D
-  Secured and is strongly authenticated (SCA in PSD2).
+  <code>card[pares]</code> is attempted 3-D Secure or fully 3-D Secure; if fully
+  3-D Secure, then therefore strongly authenticated (SCA in PSD2).
```

Comment:
Suggestions:

* s/includes/includes a valid/ (I admit "valid" is very vague, but I will refrain from using e.g. "cryptographically valid", "with a valid signature", "with valid transaction details", "with details matching the authorization being created", etc.
* s/, then therefore/then/

Reply:
* It's implicit that it's valid, because non-valid (and also non-authenticated 3DS) will not result in an approved authorisation 🙂
* I'm the typo champ. Sorry. Will fix. Thanks 🙂

Reply:
@mt-clearhaus, re. the first bullet, WDYT?

Reply:
I agree that "approved" in the sentence makes "valid" superfluous. I had not thought about that.

Reply:
Then we better explain. Also, this relates to sending non-authenticated transactions with UCAF/CAVV which we will eventually do. I'll make it more explicit.